crates/plugins/src/manifest.rs

```diff
@@ -38,6 +39,15 @@ impl PluginManifest {
     pub fn license(&self) -> &str {
         self.license.as_ref()
     }
+
+    pub fn description(&self) -> Option<&str> {
+        self.description.as_deref()
+    }
+
+    pub fn homepage_url(&self) -> Option<Url> {
+        Url::parse(&self.homepage).ok()
```

Comment:
Looks like the `?` is missing:

```suggestion
        Url::parse(&self.homepage?).ok()
```

Reply:
Will need to solve this locally. With `?` it was also failing.

```
error[E0507]: cannot move out of `self.homepage` which is behind a shared reference
  --> crates/plugins/src/manifest.rs:48:21
   |
48 |         Url::parse(&self.homepage?).ok()
   |                     ^^^^^^^^^^^^^-
   |                     |
   |                     `self.homepage` moved due to this method call
   |                     move occurs because `self.homepage` has type `std::option::Option<std::string::String>`, which does not implement the `Copy` trait
   |
note: `std::ops::Try::branch` takes ownership of the receiver `self`, which moves `self.homepage`
  --> /rustc/9eb3afe9ebe9c7d2b84b71002d44f4a0edac95e0/library/core/src/ops/try_trait.rs:2[18](https://github.com/fermyon/spin/actions/runs/4979090393/jobs/8910255584#step:5:19):15
help: you can `clone` the value and consume it, but this might not be your desired behavior
   |
48 |         Url::parse(&clone().self.homepage?).ok()
   |                     ++++++++
```

Reply:
Just need good old `as_deref`.